lecture_2/homework/app/models/warrior.rb

```diff
@@ -0,0 +1,13 @@
+class Warrior < ApplicationRecord
+  belongs_to :clan
+  belongs_to :defensible, polymorphic: true
+  has_one :weapon
+
+  validates :name, presence: true
```

Comment:
Tutaj z kolei dodałeś unikalny indeks na bazie danych ale nie walidujesz unikalności w modelu 😢 To też będzie powodować nieobsłużone błędy